run.ts

```diff
@@ -4,19 +4,18 @@ import { PromiseQueue } from "./deps/pqueue.ts"
 import { Browser } from "./deps/puppeteer.ts"
 import { deferred } from "./deps/std/async.ts"
 import { Buffer, readLines } from "./deps/std/io.ts"
-import * as path from "./deps/std/path.ts"
 import { readerFromStreamReader, writeAll } from "./deps/std/streams.ts"
 
 export interface RunOptions {
-  paths: (readonly [dir: string, fileName: string])[]
+  paths: readonly string[]
```

Comment:
Does this need to be `readonly`?

Reply:
doesn't have to be. Just made it immutable to prevent the "outside world" from modifying the results -- but at the same time, no other part of the code is going to edit the results. its about as semantically the same as should you mark all variables `final` in Java.